docs/NEXT_QUESTION_FEATURE.md

```
@@ -0,0 +1,54 @@
+
+### Query Hook
+
+**`useNextUnansweredQuestion()`**
+
+- React Query hook that fetches the next unanswered question ID
+- Automatically refetches when user submits an answer
+- Returns `{ questionId: string | null, remainingQuestions: number }`
+
+### UI Changes
+
+**IndividualQuestionPage Component**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)